Backend/UnitTesting/App.config

```diff
@@ -1,25 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file should not have changed

Reply:
There is actually no difference here, I don't know why it is complaining...